ProjectSettings/ProjectSettings.asset

```diff
@@ -521,7 +521,7 @@ PlayerSettings:
     iOS: 1
   incrementalIl2cppBuild: {}
   additionalIl2CppArgs: 
-  scriptingRuntimeVersion: 0
+  scriptingRuntimeVersion: 1
```

Comment:
> [.NET 4.6] framework will become the default in a forthcoming version of Unity.

Is this documented somewhere? I know the default is still .NET 3.5 in Unity 2017.2.

Prior to this change, NuGetForUnity only required .NET 3.5 and could be installed and used in a Unity 5.2 or later project.  With this change, the entire Unity 5.x line would be excluded as users would need Unity 2017.1 or later. Are we really okay with excluding those users?

Sidenote: I wish there was an official Unity version usage chart that showed where the majority of users were.  That'd really help with making decisions such as these.

Reply:
[This post](https://forum.unity.com/threads/future-plans-for-the-mono-runtime-upgrade.464327/) suggests soon, but apparently it wasn't ready for 2017.2. You raise a good point. Since this isn't the default I'll revert the runtime version update. The authentication feature won't work by default, but anyone who needs it can switch their project to 4.6.

Reply:
From my experience installing packages had no issues being on .NET 3.5. Though, it was required for getting packages restored.

Reply:
When I pass authentication information, I have an issue on .NET 3.5 that seems to be fixed in .NET 4.6. I have reverted my change to the .NET version and added an error message to inform folks who experience the authentication issue that they may need to update to 4.6.

Reply:
@GlitchEnzo Please take a look at my latest commit when you get a chance. Should be quick to review.